src/test/java/org/myrobotlab/codec/CodecUtilsTest.java

```diff
@@ -24,27 +25,106 @@ public void testLocale() {
         code = "-";
         mrlLocale = new Locale(code);
         json = CodecUtils.toJson(mrlLocale);
-        assertEquals("{\"language\":null,\"displayLanguage\":null,\"country\":null,\"displayCountry\":null,\"tag\":null,\"class\":\"org.myrobotlab.service.data.Locale\"}", json);
+        if(!CodecUtils.USING_GSON) {
+            assertEquals("{\"language\":null,\"displayLanguage\":null,\"country\":null,\"displayCountry\":null,\"tag\":null,\"class\":\"org.myrobotlab.service.data.Locale\"}", json);
```

Comment:
USING_GSON=true fail these tests ?

Reply:
Gson by default does not serialize null values while Jackson does, I think serializing nulls is probably best since we use it in a lot of places as a special value, and I could probably configure gson to do that

Reply:
I've made GSON serialize nulls as well so this test now doesn't need to differentiate between the serializers